x-pack/plugins/security_solution/public/actions/filter/cell_action/filter_in.test.ts

```diff
@@ -46,7 +46,7 @@ jest.mock('@kbn/ui-actions-plugin/public', () => ({
 }));
 
 const { storage } = createSecuritySolutionStorageMock();
-const mockStore = createStore(mockState, SUB_PLUGINS_REDUCER, kibanaObservable, storage);
+const mockStore = createStore(mockState, SUB_PLUGINS_REDUCER, kibanaMock, storage);
```

Comment:
One thought/suggestion: since you're already touching all of these tests, now might be a good time to swap this usage of `createStore` with a simpler `createMockStore`. It seems like the majority of tests don't use these dependencies directly, and the number of changes here are indicative (to me) of a need for some kind of abstraction.

Reply:
@rylnd that is a great idea. Will do that!

Reply:
✅ https://github.com/elastic/kibana/pull/175962/commits/4580077e8a33538ce4737d2c3c6cee9284e8852f